app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -3,11 +3,11 @@
 
   <% if follow_request %>
     <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request,  method: :delete, remote: true, class: "#{dom_id(follow_request)} btn btn-outline-secondary" do %>
         Un-request
       <% end %>
     <% elsif follow_request.accepted? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request, method: :delete, remote: true, class: "#{dom_id(follow_request)} btn btn-outline-secondary", remote: true do %>
```

Comment:
Remember that you can format the link_to methods with the commas. New lines made it a bit easier for me. Good job Sera!